braidpool_spec.md

```diff
@@ -533,6 +805,20 @@ can be delivered to an exchange, market maker, or OTC desk in exchange for
 immediate payment (over Lightning, for example) or as part of a derivatives
 contract.
 
+[kp]
+
+We need a commitment to buy the rewards. I have addressed this in my blog post
+and proposed a solution that works with channels construction.
+
+What we need issue
+
+1. Market maker to put in BTC to be redeemed for all BTC that will be generated
+   by a certain miner.
+2. At the end of the period, neither MM nor M should be able to pull out of the
+   transaction.
+
+[kp]
+
```

Comment:
This is part of market design. As we don't want to design a market, we don't have to solve atomic swaps or guarantee that this (private) contract is executed.